engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -173,9 +173,10 @@ private void loadModulesFromClassPath() {
                     continue;
                 }
                 if (registry.add(module)) {
-                    logger.info("Loaded {} from {}", module.getId(), path);
+                    logger.atInfo().addArgument(() -> module.getId()).addArgument(() -> path).log("Loaded {} from {}");
```

Comment:
simple getter method so impact neglectable
```suggestion
                    logger.info("Loaded {} from {}", module.getId(), path); //NOPMD
```